readme.md

```diff
@@ -85,6 +85,36 @@ Default: `false` *(No localization)*
 
 **Note:** Localization should generally work in browsers. Node.js needs to be [built](https://github.com/nodejs/node/wiki/Intl) with `full-icu` or `system-icu`. Alternatively, the [`full-icu`](https://github.com/unicode-org/full-icu-npm) module can be used to provide support at runtime. [Node.js 13](https://nodejs.org/en/blog/release/v13.0.0/) and later ships with ICU by default.
 
+##### minimumFractionDigits
+
+Type: `number`\
+Default: `undefined`
+
+The minimum number of fraction digits to display.
+
+```js
+const prettyBytes = require('pretty-bytes');
+
+// Show number with at least 3 fractional digits
```

Comment:
index.d.ts and readme should be in sync.

Reply:
Done.